arlas-core/src/main/java/io/arlas/server/app/ArlasServerConfiguration.java

```diff
@@ -170,6 +170,13 @@ public void check() throws ArlasConfigurationException {
         if (arlasServiceExploreEnabled == null) {
             arlasServiceExploreEnabled = true;
         }
+        if (inspireConfiguration == null) {
+            inspireConfiguration = new InspireConfiguration();
+            inspireConfiguration.enabled = false;
+            inspireConfiguration.publicAccessLimitations = "no limitations apply";
+            inspireConfiguration.accessAndUseConditions = "no conditions apply";
+            inspireConfiguration.servicesDateOfCreation = "2018-11-27";
```

Comment:
Strange to have a hardcoded date here.
Maybe create a date from `LocalDate.now()`?

Reply:
It corresponds to the date we released v9.6.0 that supports Inspire compliant services